attestation/commandrun/commandrun.go

```diff
@@ -154,21 +225,46 @@ func (r *CommandRun) runCmd(ctx *attestation.AttestationContext) error {
 		enableTracing(c)
 	}
 
+	runtime.LockOSThread()
+
 	if err := c.Start(); err != nil {
 		return err
 	}
 
-	var err error
+	if r.tetragonAddress != "" {
+		syscall.Kill(c.Process.Pid, syscall.SIGSTOP)
+		tc, err := NewTC(ctx, r, c.Process.Pid)
+		if err != nil {
+			return err
+		}
+		err = tc.Start()
+		if err != nil {
+			return err
+		}
+		defer tc.Stop(r)
+		time.Sleep(time.Second * 1)
```

Comment:
Similarly we will need to iron out any timing issues to make sure this is all dependable and not reliant on inconsistent timings.